Makefile

```diff
@@ -0,0 +1,80 @@
+.DEFAULT_GOAL := help
+
+CURRENT_VENV := $(shell python -c 'from __future__ import print_function; import sys; print(sys.prefix if hasattr(sys, "real_prefix") or (hasattr(sys, "base_prefix") and sys.base_prefix != sys.prefix) else "", end="")')
+
+ifeq ($(CURRENT_VENV),)
+VIRTUALENV := .venv
+else
+VIRTUALENV := $(CURRENT_VENV)
+endif
+
+PIPENV := PIPENV_VENV_IN_PROJECT=1 pipenv
+WITH_PIPENV := $(PIPENV) run
+
+help:  ## print this help
```

Comment:
I don't know much about makefiles, but don't you also need to mark `help` as being `.PHONY`? It is in the link below.